.github/workflows/ci.yml

```diff
@@ -0,0 +1,56 @@
+name: CI
+
+on:
+  push:
+
+jobs:
+  build:
+    strategy:
+      # The matrix will run all possible combinations in parallel.
+      matrix:
+        container: [ ubuntu-latest, macos-latest ]
+
+    runs-on: ${{ matrix.container }}
+
+    steps:
+      - uses: actions/checkout@v2
+
+      # Use Go 1.17
+      - name: Switch to Go 1.17
+        run:
+          echo "$GOROOT_1_17_X64"/bin >> $GITHUB_PATH
+
+      - name: Go dependencies
+        shell: bash
+        run: |
+          make go_dependencies
+          echo "$HOME/go/bin" >> $GITHUB_PATH
+
+      - name: Toit dependencies
+        id: toit
+        shell: bash
+        run: |
```

Comment:
you could preferable make two steps and then do the if in yaml instead.

Reply:
I actually prefer it this way. I had it differently first, and this feels more natural.